### PR TITLE
Switch Mac project to .net6

### PR DESCRIPTION
**Pull request by BretJohnson**

### CODE REVIEW

**Comment by BretJohnson** [comment]
This PR branch is referenced from https://github.com/xamarin/UITools/pull/608

**Review by Therzok — APPROVED** [review]

**Review by drasticactions — APPROVED** [review]

File: .build/automation/stages/validate.yml [hunk 1]
```diff
@@ -12,6 +12,19 @@ stages:
             clean: true
             persistCredentials: true
 
+          - task: CmdLine@1
```
Comment (mcumming) [inline .build/automation/stages/validate.yml:15]: Why is this required? You are essentially setting the value back to itself?

**Review by mcumming — CHANGES_REQUESTED** [review]

File: .build/automation/stages/validate.yml (hunk 1 above)
Comment (BretJohnson) [reply .build/automation/stages/validate.yml:15]: You are asking about the Set Token step? I'm not sure why (or if) it's needed, but we seem to do everywhere before calling the provisionator. I copied it from the Windows section above. It's also in Hot Reload here: https://github.com/xamarin/UITools/blob/a90b49522b5aadc38c8f63992cf80c1e63d60e21/HotReload/build/automation/stages/validate.yml#L189
Do think it doesn't actually do anything? It seems like it's setting the GITHUB_TOKEN environment variable from the pipeline property.

Comment (BretJohnson) [reply .build/automation/stages/validate.yml:15]: @mcumming ^^^

Comment (BretJohnson) [reply .build/automation/stages/validate.yml]: Indeed, this wasn't needed. I removed it, from both Mac and Windows.

**Review by mcumming — APPROVED** [review]

**Comment by Therzok** [comment]
Have we tried inserting this in VSMac?
